plugins/scala_test/README.md

```diff
@@ -0,0 +1,26 @@
+# `scalatest` Migration Plugin 
+
```

Comment:
This needs a description/explanation of what this is, before the Usage instructions.

Would also be a good point to note if this is a WIP or already functional and for which cases.